besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -949,6 +960,17 @@ private void addConfigurationService() {
     }
   }
 
+  /* NOTE: Must be called after addConfigurationService */
+  private void addNodeKey() {
```

Comment:
I'd put this down the some of the storageService commands - and wondering if its more of a "construct" rather than an "add"

Reply:
removed.